libs/core-wrappers/src/lib/Navbar/Navbar.tsx

```diff
@@ -4,24 +4,27 @@ import { Icon } from '@tacc/core-components';
 import styles from './Navbar.module.css';
 
 export const NavItem: React.FC<
-  React.PropsWithChildren<{ to: string; icon?: string }>
-> = ({ to, icon, children }) => (
-  <NavLink
-    to={to}
-    className={(isActive) => (isActive ? styles['active'] : styles['nav-link'])}
-  >
-    <div className={styles['nav-content']}>
-      {icon && <Icon name={icon} />}
-      {/* we'll want to set name based on the app */}
-      <span className={styles['nav-text']}>{children}</span>
-    </div>
+  React.PropsWithChildren<{ to: string; icon?: string; end?: boolean }>
+> = ({ to, icon, end, children }) => (
+  <NavLink to={to} end={end} className={styles['nav-link']}>
+    {({ isActive }) => (
+      <div
+        className={`${styles['nav-content']} ${
+          isActive ? styles['nav-active'] : ''
+        }`}
+      >
+        {icon && <Icon name={icon} />}
+        {/* we'll want to set name based on the app */}
+        <span className={styles['nav-text']}>{children}</span>
+      </div>
+    )}
   </NavLink>
 );
 
 type NavbarProps = React.PropsWithChildren<unknown>;
 
 const Navbar: React.FC<NavbarProps> = ({ children }) => {
-  return <div className={`${styles['nav-list']}`}>{children}</div>;
```

Comment:
If you're dropping this class, then go ahead and drop [the two respective styles](https://github.com/TACC/tup-ui/blob/task/TUP-317--sidebar-style/libs/core-wrappers/src/lib/Navbar/Navbar.module.css#L5-L6) too.